### ISSUE TITLE
Is it possible to hide the percentage

### ISSUE THREAD

**Issue by geo80** [marbl/Krona#30]
Is it possible to show the last level without its percentage?

**Comment by ondovb** [comment]
No, but this would be an easy option to add in the next version. In the meantime you would have to save a snapshot and delete the percentages with an SVG editor.

**Comment by geo80** [comment]
Indeed (for the SVG editor solution) Thanks!
